src/main/java/com/denizenscript/depenizen/bukkit/commands/mythicmobs/MythicThreatCommand.java

```diff
@@ -42,73 +42,33 @@ public MythicThreatCommand() {
-                scriptEntry.addObject("mythicmob", arg.asType(MythicMobsMobTag.class));
-            }
-            else if (!scriptEntry.hasObject("operation")
-                    && arg.matchesEnum(Operation.class)) {
-                scriptEntry.addObject("operation", arg.asElement());
-            }
-            else if (!scriptEntry.hasObject("threat")
-                    && arg.matchesFloat()) {
-                scriptEntry.addObject("threat", arg.asElement());
-            }
-            else if (!scriptEntry.hasObject("targets")
-                    && arg.matchesPrefix("for")
-                    && arg.matchesArgumentList(EntityTag.class)) {
-                scriptEntry.addObject("targets", arg.asType(ListTag.class).filter(EntityTag.class, scriptEntry));
-            }
-            else {
-                arg.reportUnhandled();
-            }
-        }
-        if (!scriptEntry.hasObject("mythicmob")) {
-            throw new InvalidArgumentsException("Must specify a MythicMob!");
-        }
-        if (!scriptEntry.hasObject("operation")) {
-            throw new InvalidArgumentsException("Must specify an Operation!");
-        }
-        if (!scriptEntry.hasObject("threat")) {
-            throw new InvalidArgumentsException("Must specify a threat value!");
-        }
-        if (!scriptEntry.hasObject("targets")) {
-            scriptEntry.defaultObject("targets", (Utilities.entryHasPlayer(scriptEntry) ? Collections.singletonList(Utilities.getEntryPlayer(scriptEntry).getDenizenEntity()) : null));
-        }
-    }
+    public enum Operation {ADD, SUBTRACT, SET}
 
-    @Override
-    public void execute(ScriptEntry scriptEntry) {
-        MythicMobsMobTag mythicmob = scriptEntry.getObjectTag("mythicmob");
-        ElementTag operation = scriptEntry.getObjectTag("operation");
-        ElementTag threat = scriptEntry.getElement("threat");
-        List<EntityTag> targets = (List<EntityTag>) scriptEntry.getObject("targets");
-        if (scriptEntry.dbCallShouldDebug()) {
-            Debug.report(scriptEntry, getName(), mythicmob, operation, threat, db("targets", targets));
-        }
+    public static void autoExecute(ScriptEntry scriptEntry,
+                                   @ArgLinear @ArgName("mythicmob") MythicMobsMobTag mythicmob,
+                                   @ArgLinear @ArgName("operation") Operation operation,
+                                   @ArgLinear @ArgName("threat") ElementTag threat,
+                                   @ArgPrefixed @ArgName("for") @ArgDefaultNull ListTag targets) {
         if (!mythicmob.getMob().hasThreatTable()) {
             Debug.echoError("MythicMob does not have a threat table: " + mythicmob);
             return;
         }
-        Operation op = Operation.valueOf(operation.toString().toUpperCase());
-        switch (op) {
+        if (targets == null) {
+            targets = new ListTag(Utilities.getEntryPlayer(scriptEntry).getDenizenEntity());
```

Comment:
This will error if there is no linked player, thus the check in the original version of the `targets` default.